packages/vscode-extension/src/suitecloud.ts

```diff
@@ -19,10 +20,16 @@ import { installIfNeeded } from './core/sdksetup/SdkServices';
 const SCLOUD_OUTPUT_CHANNEL_NAME = 'SuiteCloud';
 
 function register<T extends BaseAction>(command: string, action: T) {
-	return vscode.commands.registerCommand(command, () => action.run());
+	return vscode.commands.registerCommand(command, (uri?: vscode.Uri) => {
+		if (uri && uri.fsPath) {
+			action.run(uri.fsPath);
+		} else {
+			action.run();
+		}
+	});
```

Comment:
This is a question more than a comment. Who is calling this callback? I mean who is the caller of function of the second argument of the reisterCommand, or when is it actually executed and what are the possible arguments passed there.
I guess you can get from there some info for ImportFiles but I'm a little bit lost here.

Reply:
Yeah, I understand it is not clear. I'm adding a comment to it.
Basically, when you are calling an import from a context menu, it comes with an uri, with the info of the file you are right-clicking.
Meanwhile... if the command is called from the command palette.. it doesn't receive any uri. 
I put this if-else to avoid NullPointer excpetions with uri while only passing the fspath